Libplanet/Net/Swarm.cs

```diff
@@ -590,6 +590,25 @@ await DialToExistingPeers(dialTimeout, cancellationToken)
                 cancellationToken);
         }
 
+        /// <summary>
+        /// Validates all <see cref="Peer"/>s in the routing table by sending a  simple message.
```

Comment:
Is there a reason to require two spaces?

```suggestion
        /// Validates all <see cref="Peer"/>s in the routing table by sending a simple message.
```

Reply:
No, I'll fix it. :)